### PR TITLE
Feat: Added rule for manta tumor only calling. Refactor of old rule

### PR DESCRIPTION
**Pull request by jonca79**

### CODE REVIEW

File: workflow/rules/manta.smk [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    resources:
+        mem_mb=config.get("manta_run_workflow_tn", {}).get("mem_mb", config["default_resources"]["mem_mb"]),
+        mem_per_cpu=config.get("manta_run_workflow_tn", {}).get("mem_per_cpu", config["default_resources"]["mem_per_cpu"]),
+        partition=config.get("manta_run_workflow_tn", {}).get("partition", config["default_resources"]["partition"]),
+        threads=config.get("manta_run_workflow_tn", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("manta_run_workflow_tn", {}).get("time", config["default_resources"]["time"]),
+    container:
+        config.get("manta_run_workflow_tn", {}).get("container", config["default_container"])
+    conda:
+        "../envs/manta_run_workflow_tn.yaml"
```
Comment (marrip) [inline workflow/rules/manta.smk:79]: Should be `../envs/manta.yaml`

File: workflow/rules/manta.smk [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    resources:
+        mem_mb=config.get("manta_run_workflow_t", {}).get("mem_mb", config["default_resources"]["mem_mb"]),
+        mem_per_cpu=config.get("manta_run_workflow_t", {}).get("mem_per_cpu", config["default_resources"]["mem_per_cpu"]),
+        partition=config.get("manta_run_workflow_t", {}).get("partition", config["default_resources"]["partition"]),
+        threads=config.get("manta_run_workflow_t", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("manta_run_workflow_t", {}).get("time", config["default_resources"]["time"]),
+    container:
+        config.get("manta_run_workflow_t", {}).get("container", config["default_container"])
+    conda:
+        "../envs/manta_run_workflow_t.yaml"
```
Comment (marrip) [inline workflow/rules/manta.smk:153]: Should be `../envs/manta.yaml`

File: workflow/rules/manta.smk (hunk 1 above)
Comment (jonca79) [reply workflow/rules/manta.smk]: done

File: workflow/rules/manta.smk (hunk 2 above)
Comment (jonca79) [reply workflow/rules/manta.smk]: done

**Review by marrip — APPROVED** [review]

**Pull request merged**